core/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+#![recursion_limit = "256"]
```

Comment:
Just curious, what macro expansion made this needed?

Reply:
I kept it because it was there originally. Iirc this was introduced when we were using the select! macro for the event based orderbook. I doubt it's needed anymore.

Reply:
Oh my bad, I thought it was added here!

Reply:
I would just leave it as is then.